### DISCUSSION TITLE
심화프로젝트 3주차 팀회고

### DISCUSSION THREAD

**Discussion by sulyoungoh** [codeit-bootcamp-frontend/3-Weekly-Mission#347] [회고]
심화 프로젝트 3주차 팀 회고를 하고([회고 방식 상세보기](https://github.com/codeit-bootcamp-frontend/3-Weekly-Mission/discussions/346#discussion-6517270)), 이전처럼 댓글 남겨주세요🙏

아래 내용들을 토대로 이번 팀 회고에서 점검해보면 좋을 것 같아요.
마지막 프로젝트인만큼 꼼꼼하게 기록하여 나와 팀원들의 모습을 남겨보세요 ☺️
- 이번주 나와 팀원들은 무엇을 잘 했는지, 어떤 문제를 겪었고 어떻게 해결했는지, 문제를 통해 배운 점
- 팀 소통 방식 
- 협업 관계에서 느낀 감정 
- 기타 등등

**Comment by jinahp** [comment]
# 6팀

## K (Keep)
- 코어타임, 멘토링 등 참여못할시 사전에 양해를 구하는것이 좋았다.
- 프로젝트 진행 중 막히는 게 있으면 질문을 하고, 질문을 받았을 때 답변을 해주는 팀워크가 좋은 것 같다.
- 코어 타임 이외에도 열정들이 넘쳐 소통에 불편함이 없는 것이 좋았다.
- 코어 타임 이후의 시간에도 다들 새벽까지 모여서 얘기 나누고 해결해나가는 모습이 너무 좋았습니다 !!
- 모각코 얘기를 했었는데 바로 진행이 되어서 좋았습니다 ㅎㅎ

## P (Problem)
- 팀원 PR에 대한 코드리뷰가 부족한것 같다.
- 컨벤션이 잘 지켜지지 않는 듯하다.
- 코드리뷰시 코드에 대한 이야기도 하면 좋을 것 같다.
- 저도 코드리뷰가 부족한 것 같아 조금 아쉬웠습니다!

## T (Try)
- 아직 공부 못한 부분이라 봐도 이해 안되는게 대부분이지만, 최대한 이게 뭘 하는 코드인지 정도는 이해하려고 노력해보자.
- 최대한 정했던 컨벤션을 따라가려고 노력하지만 조금 더 신경써보면 좋을 듯싶다. 그 외엔 💯
- 5월이 되면 중간 중간 어느정도 진행이 됐는지 서로 보고 하는 것도 좋을 것 같다.
- 바쁠 시기이지만 코드리뷰를 통해 얻어가는 것도 많다고 생각하기 때문에 코드에 대한 얘기를 나눠보는게 좋을 것 같습니다!

**Comment by yunsunglee2** [comment]
# 7팀

## K (Keep)
- 프로젝트 현황을 기록하는 일
- 컴포넌트명, 변수명 수정할 부분이 생기면 즉시하기 
- 팀원과 겹치는 부분 있으면 바로 수정하기
- jsDocs 열심히 쓰기 

## P (Problem)
- 필수 구현 사항, 부가적 기능을 나누어 개발을 진행하다 보니 진행이 늘어지는 경향이 조금 있다.
- 적극적인 팀활동 참여에 대한 아쉬움 

## T (Try)
- 프로젝트 완성을 위한 체력관리
- 기획한 서비스에 걸맞는 특색있는 기능에 대해서 생각 
- 서비스 유저플로우를 생각해보며 놓친 부분이 있는지 검토 
- 맡은 부분 이외에 추가적인 기능 구현에 대해서 생각
- 큰 일정 틀을 잡고 마감기한 정하기

**Comment by Dev-Duke-Seo** [comment]
# K (Keep)
- 모두 본격적인 작업을 시작하고 열심히 임하는 것 같아 좋습니다.
- 다들 각자의 파트에서 최선을 다하고 있는 것 같아서 좋습니다.
- 어려운 부분이 있으면 다같이 도와주는 부분이 좋습니다.
- 각자 파트 시작전에 페어프로그래밍의 코드 방식을 맞춰보는 부분이 좋았습니다.
- 문제가 생기면 본인의 일 처럼 도와주는 모습이 좋습니다.
- 각 파트별로 소통이 잘 되는 것 같습니다.
- 코드 리뷰를 꼼꼼하게 해줘서 부족한 부분을 고치는데 많은 도움이 됩니다.
- 정해둔 컨벤션을 잘 따라가려고 노력하고 또 서로 고쳐주는 부분이 좋았습니다.

# P (Problem)
- 아직 기획이 안된 사소한 부분들이 많습니다.
- 의견 모으기가 쉽지 않은 것 같습니다.
- PR 리뷰를 제 시간에 남기는게 어려운 것 같습니다.
- 작업 속도가 생각보다 느린 것 같아 걱정이 됩니다.

# T (Try)
- 파트별로 소통이 더 늘어야할 것 같습니다.
- 디스코드를 활용하여 소통을 활발하게 하면 좋을 것 같습니다.
- PR 리뷰를 안 미루도록 노력해봐야 할 것 같습니다.
- PR 리뷰를 하루 안에 끝마치도록 노력합시다!!!!!!!!!!

**Comment by Song-Sang** [comment]
## K (Keep)
- 피그마 시안 기본 기능 구현은 계획한대로 중간 제출 전까지 어느정도 마무리 되어가는거같아 좋습니다.
- 초기 R&R이 잘 되어서 진행하면서 별다른 상의 없이도 계획대로 잘 진행되어가고 있습니다.
- 깃 허브 활용한 일정관리가 잘 진행되고 있어서, 팀원들간 진행 상태 확인하기가 좋습니다.
- 맡은 부분 빨리 끝나신 분들을 알아서 남은 부분 확인해서 진행해 주고 계신점이 좋습니다.


## P (Problem)
- R&R 분배 문제와 별개로 팀원 별로 진행 속도 차이가 조금 있습니다.
- 제시된 기능 구현이 어느정도 완성 되어가서 추가 기능에 대한 상의가 필요합니다.
- 급하게 일정을 진행하느라 중복된 코드들이 몇가지 있는데 상의 후 리팩토링이 필요합니다.
- 초기에 활발하게 진행되던 PR 과 PR리뷰가 점차 느려지고 있는 문제,

## T (Try)
- 5/1 일 중간 제출 이후 팀미팅을 통해 추가 기능 및 리팩토링에 관한 회이 진행 예정.
- 5/1 일 전까지 각자 맡았던 기능들에 대한 리팩토링 및 추가 기능에 대한 의견 준비하기.
- PR과 PR리뷰에 대한 관심 다시 한 번 상기하기.

**Comment by eundin** [comment]
# 2팀

## K (Keep)
- 저번주보다 궁금한 부분에 대한 공유가 잘 되고있는것같습니다!
- 다들 열심히 하고 계신 것 같다.
- 예기치 못한 일정이 생겨서 참여하지 못할시이 미리 얘기하는게 좋았습니다.

## P (Problem)
- 점점 딜레이되는 일정. 여유가 있다는 인식때문인지 그 부분이 조금 아쉽습니다.
- 예상 소요 기간이 조금 길어지는 것 같다.
- 팀원들의 코드 리뷰가 좀더 활발히 이루어지면 좋겠습니다.
- PR리뷰를 좀 더 상세히 적어주시면 좋겠습니다.

## T (Try)
- 모르는 부분을 서로 공유하며 같이 해결해나가면 좋을 것 같다.
- 프로젝트 완성을 위해서 각자 바디컨디션 잘 챙겼으면 좋겠습니다.
- 사소한 거라도 모르는 부분이나 궁금한 점이 있으면 서로 공유하면 좋겠습니다.

**Comment by ohddang** [comment]
# 3팀

## K (Keep)
- git pr 조금씩 개선되고 있음
- 어려운 문제에 도전하고 해결해냄
- 이대로만 갑시다
- 각자 맡은 부분 열심히 하심

## P (Problem)
- 프론트쪽 더 빨리했으면함
- 일단 작은 기능부터 구현해보기
- 소통이 좀 부족한듯함
- 이번주 안에 기본 기능 다 할 수 있을 지 고민
- 시간이 조금 부족하다 느낌, 브랜치 전략
- 잘하고 있는건지 모르겠음

## T (Try)
- 마무리인 만큼 단순 기능뿐만이 아닌 깊이 있는 코드를 작성하기 위해 노력해보자
- git issue에 todo 지속적으로 체우기
- 프론트엔드와 백엔드를 같이 진행할 수 있게 하자
- 일정 쪼개서 시간관리 더 잘해야할 듯 함
- 다른조 디자이너랑 소통해보기
